src/Turbo/Attribute/Broadcast.php

```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\UX\Turbo;
+namespace Symfony\UX\Turbo\Attribute;
```

Comment:
let's make this a convention really - we need a dedicated namespace for attributes